### ISSUE TITLE
Staging Review finding: Cognitive load concerns around a muddled critical action visual hierarchy

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#106797]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** AEDP
**Product:** Design Patterns
**Feature:** Critical Information and Service List Item Components

## Findings details
**VA.gov Experience Standard - issue:** [User's experience is consistent with the VA.gov design principles.](https://design.va.gov/about/experience-standards/#users-experience-is-consistent-with-the-vagov-design-principles)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility), @allison0034 (Design)

### Description

From the [VA design principles](https://design.va.gov/about/principles#simple):
> **Focus users on the primary task, free of distractions.** Break complex tasks into small logically ordered steps. Provide context, explain impacts, and use VA sources to reduce data entry where possible.

This is a tremendously important principle for reducing cognitive load. When a user lands on a page, it should be clear what the purpose of the page or what the next step in a user flow should be. For sighted users, this will largely match the visual hierarchy of the page. Attention is expensive, and asking for a user's attention too many times has diminishing returns --- as we've seen this in the alert fatigue that's been reported in user research.

The critical action component seems to be intended to help with this, drawing attention to the most important task with a particularly bright color and bold text. To me, this is comparable to using a brighter highlighter color to differentiate from all of the other highlighter markings on the page. I don't think it's a solution that really engages with the core problem.

I also think that intention falls short when multiple critical actions are displayed on a page, as in the [MyVA example where they're stacked](https://dev-design.va.gov/3973/components/critical-action#accessibility-considerations:~:text=currently%20in%20production.-,On%20the%20My%20VA%20page,-%2C%20you%20can%20stack) or in situations where multiple cards or service list items have a critical action. The guidance doesn't provide any direction on how VFS teams should prioritize multiple critical actions or if there's an upper limit. And as a user looking at the critical actions on the page, the visual hierarchy doesn't point you toward a primary task or provide any logical ordering beyond top-to-bottom positioning.

All of this also takes place in the context of pages that may also have alerts and other attention-grabbing components. If I have an error alert at the top of the page and three critical actions in three different cards, what should I address first? I don't know that the answer is going to be clear to the user.

<details><summary>Link, screenshot or steps to recreate</summary>

![Five critical actions stacked at the top of MyVA. Each critical action has the same visual weight.](https://dev-design.va.gov/3973/images/components/critical-action/critical-action-in-my-va.png)

</details>

## Recommended action

I recognize that one component can't solve all of VA.gov's visual hierarchy problems, and that this component is intended to solve problems in specific situations. But I do have a few recommendations I hope you'll explore:
- Consider a secondary variant of the critical action component that's less visually intrusive, eg. a lighter background color and/or other design elements that help a user perceive it as less important than the primary variant. When multiple critical actions appear on the page, consider guidance that only one critical action should use the primary variant and the rest should use the secondary variant --- similar to existing guidance around primary and secondary buttons. Once the highest-priority critical action is addressed, then when the user returns to the page the next-highest priority can use the primary variant, and so on.
- Consider guidance that limits the number of critical actions that may be displayed on a single page.
- Consider guidance that limits the number of critical actions combined with the number of alerts on the page (eg. no more than 3 alerts and critical actions all together or similar).
- Consider content guidance that requires critical action link text to include a deadline and/or a description of the negative consequences of not taking the action, so users have enough information to determine their priorities.
- Consider providing guidance on how teams with competing priorities can triage their critical actions when feeding critical actions to a shared space (eg. providing guidance for how critical actions on MyVA can be prioritized when the actions point to products owned by different teams.)

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by msbtterswrth** [comment]
@rtwell we'd love to get your thoughts on this feedback when you've got some time.

**Comment by rtwell** [comment]
Your considerations will be considered 😉. Please see [feedback in Issue 106793](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106793#issuecomment-2783921759) which addresses some of this.
